lang/python: python host packages installed to usr/lib/ with installer 1.0.0, breaking subsequent host builds

### Package Name

python-installer

### Maintainer

@commodo

### OpenWrt Version

SNAPSHOT

### OpenWrt Target/Subtarget

x86/64

### Steps to Reproduce

1. Build any package that depends on python-installer/host, e.g.:
   make package/feeds/packages/python-packaging/host/compile V=s

2. Error:
   /staging_dir/hostpkg/bin/python3.14: No module named installer

3. Diagnosis:
   find staging_dir/hostpkg -path "*/installer/__init__.py"
   → staging_dir/hostpkg/usr/lib/python3.14/site-packages/installer/__init__.py

   python3.14 -c "import sysconfig; print(sysconfig.get_path('purelib', vars={'base':'','platbase':''}))"
   → /lib/python3.14/site-packages

   installer 1.0.0 installs to usr/lib/ but Python looks in lib/ (no usr/ prefix),
   causing all subsequent Python host package builds to fail with ModuleNotFoundError.

4. Root cause: installer 1.0.0 changed path computation behavior with --prefix "".
   Not reproducible with installer 0.7.0.

### Actual Behaviour

Build fails with `No module named installer` when building any Python host package
that depends on `python-installer/host` (e.g. python-packaging, python-setuptools).

After python-installer 1.0.0 is installed to staging_dir/hostpkg, the module is
placed under `usr/lib/python3.14/site-packages/` but the host Python interpreter
only searches `lib/python3.14/site-packages/`, so the module cannot be found.

The `usr/lib/` prefix is introduced by installer 1.0.0's new path computation with
`--prefix ""`. This behavior did not occur with installer 0.7.0.

### Confirmation Checklist

- [x] The package is maintained in this repository.
- [x] I understand that issues related to [the base OpenWrt repository](https://github.com/openwrt/openwrt/issues) or [LuCI repository](https://github.com/openwrt/luci/issues) will be closed.
- [x] I am reporting an issue for OpenWrt, not an unsupported fork.

Besides this error, i also have problem with this:

ERROR: package/feeds/packages/python-packaging [host] failed to build

In the log there's this:

`openwrt-SNAPSHOT/staging_dir/hostpkg/bin/python3.14   
bash: line 1: cd: -m: invalid option
cd: usage: cd [-L|[-P [-e]] [-@]] [dir]
make[3]: *** [Makefile:50: /home/openwrt-SNAPSHOT/staging_dir/hostpkg/stamp/.python-packaging_installed] Error 2`

> Besides this error, i also have problem with this:
> 
> ERROR: package/feeds/packages/python-packaging [host] failed to build
> 
> In the log there's this:
> 
> `openwrt-SNAPSHOT/staging_dir/hostpkg/bin/python3.14 bash: line 1: cd: -m: invalid option cd: usage: cd [-L|[-P [-e]] [-@]] [dir] make[3]: *** [Makefile:50: /home/openwrt-SNAPSHOT/staging_dir/hostpkg/stamp/.python-packaging_installed] Error 2`

The related python-* host-compile errors all point to `python-installer`.

```
ERROR: package/feeds/packages/python-pyproject-hooks [host] failed to build.
ERROR: package/feeds/packages/python-packaging [host] failed to build.
ERROR: package/feeds/packages/python-build [host] failed to build.
ERROR: package/feeds/packages/python-wheel [host] failed to build.
ERROR: package/feeds/packages/python-setuptools [host] failed to build.
```

--- a/lang/python/python3-host-build.mk
+++ b/lang/python/python3-host-build.mk
@@ -57,8 +57,7 @@ define Py3Host/Install/Installer
 		$(HOST_BUILD_DIR), \
 		-m installer \
 			--overwrite-existing \
-			--destdir "$(1)" \
-			--prefix "" \
+			--prefix "$(1)" \
 			"$(PYTHON3_HOST_BUILD_DIR)"/openwrt-build/$(PYTHON3_HOST_WHEEL_NAME)-$(PYTHON3_HOST_WHEEL_VERSION)-*.whl \
 			, \
 		$(PYTHON3_HOST_INSTALL_VARS) \

